.github/workflows/update_snapshots.yml

```diff
@@ -6,11 +6,11 @@ jobs:
   update_snapshots:
```

Comment:
Still seeing deprecation warnings when this workflow fails ([example](https://github.com/yext/search-ui-react/actions/runs/8904376570)). In the testing section of the PR description, can you list out all the different scenarios with which you tested the 3 updated workflows? We want to make sure we test each path the workflow may take to make sure there are no warnings on it. Depending on which flow the workflow takes, different actions can be triggered